docs/spl-docs/common-usecases.md

```diff
@@ -0,0 +1,239 @@
+I apologize for that. Here is the reordered list with only the 38 use cases you provided:
+
+# Common Use-Cases
+
+### 1. Identifying Potential Brute Force Attacks
+
+**Problem:** Detecting potential brute force attacks is crucial for maintaining network security. These attacks often involve repeated attempts to connect to critical services like SSH (port 22) or RDP (port 3389) from the same source IP, aiming to guess passwords and gain unauthorized access.
+
+**Solution:** To identify potential brute force attacks, a [`search`](search-command.md) command can be utilized to filter firewall logs for blocked connection attempts to SSH and RDP ports, count the attempts by source and destination IP, and highlight cases with a high number of attempts. [View full Solution](search-command.md/#use-case-example)
```

Comment:
Does this link work for you?
```
[View full Solution](search-command.md/#use-case-example)
```
It doesn't work for me unless it's one of these:
```
[View full Solution](search-command/#use-case-example)
[View full Solution](search-command#use-case-example)
```

Even with those versions, it just goes to the page (not the section), but I think that's a separate frontend issue

Reply:
This works for me `search-command#use-case-example`. It goes to the proper section.

Reply:
Ok great, let's use links like that then. It must just be an issue on my end